cypress/integration/unit/document-version.spec.js

```diff
@@ -52,6 +52,20 @@ context('Tests for KAS-1076', () => {
     cy.get('.vlc-scroll-wrapper__body').within(() => {
       cy.get('.vlc-document-card').as('docCards').should('have.length', 22);
     });
+
+    const linkedDocumentsNames = [
```

Comment:
Als ik me niet vergis, is dit een extract van de "files"-array hierboven. Je creeert hard-coded dependencies. Als iemand iets aanpast in lijnen 20-40 heeft dat invloed op hoe deze code werkt en vice versa. Probeer steeds te vermijden dat mensen onbewust dingen stuk kunnen maken. Schrijf hier liever iets als 
```
const linkedDocumentsNames = files.slice(a, b).map(f => f.newFileName);
```

Reply:
solved